lib/blockscore/collection/member.rb

```diff
@@ -0,0 +1,87 @@
+module BlockScore
+  class Collection
+    # Member of a {Collection} class
+    class Member < SimpleDelegator
+      # Initialize a new member
+      #
+      # @param parent [BlockScore::Base] parent resource
+      # @param instance [BlockScore::Base] member instance
+      #
+      # @return [undefined]
+      #
+      # @api private
+      def initialize(parent, instance)
+        @instance = instance
+        @parent = parent
+
+        super(instance)
+      end
+
+      # Save parent, set parent id, and save instance
+      #
+      # @example
+      #   # saves both unsaved person and unsaved question_set
+      #   person = Person.new(attributes)
+      #   question_set = QuestionSet.new
+      #   Member.new(person, question_set).save
+      #
+      # @return return value of instance `#save`
+      #
+      # @api public
+      def save
+        save_parent
+        send(:"#{parent_name}_id=", parent.id)
+        instance.save
```

Comment:
Bug: saved instances don't have their `id`'s added to the collections id's. 
Will add failing spec and fix. 
Also I'll look into potentially refactoring the tracking strategy of id's of embedded on the parent and newly created items added to collection.